frontend/src/views/WorkshopDetails.vue

```diff
@@ -65,12 +86,23 @@ async function saveItem (item) {
       <router-link class="btn p-3 hover-darken" :to="`/products/new?workshopId=${workshop.id}`">
         <font-awesome-icon :icon="['fas', 'plus']" class="fa-xl"/>
       </router-link>
+      <div type="button" class="btn p-3 hover-darken me-2" @click="showPopup = !showPopup">
+        <font-awesome-icon :icon="['fas', 'minus']" class="fa-xl"/>
```

Comment:
Min icoontje is beetje onduidelijk, mischien `boxes-packing` en dan bovenaan de pagina?